### PR TITLE
docs(v2): various improvements

### PR DESCRIPTION
**Pull request by lex111**
## Motivation

Just minor fixes before new release.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Preview.

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: website/docs/blog.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ id: blog
 title: Blog
 ---
 
+## Duplicate
```
Comment (slorber) [inline website/docs/blog.md:6]: typo?

**Review by slorber — APPROVED** [review]
LGTM 👍

File: website/docs/blog.md (hunk 1 above)
Comment (lex111) [reply website/docs/blog.md]: haha, tested something, removed these ones.

**Comment by slorber** [comment]
👍

**Pull request merged**